File: atv_classe1/java/python/ponto3d.py
from ponto import Ponto

class Ponto3D(Ponto):
    def __init__(self, x=0, y=0, z=0):
        super().__init__(int(x), int(y))
        self.z = z

    def get_z(self):
        return self.z

    def set_z(self, z: int):
        self.z = z

File: atv_classe1/java/Ponto3D.java
package atv_classe1.java;

public class Ponto3D extends Ponto {
    private int z;

    public Ponto3D(int x, int y, int z){
        super(x, y);
        this.z = z;
    }

    int getZ() {
        return this.z;
    }

    void setZ(int z) {
        this.z = z;
    }
}
